CHANGES

```diff
@@ -1,275 +1,32 @@
-   ssrc which they will be receiving can still use libsrtp.  Now the
-   SSRC value is gleaned from the rtp header and exored into the
-   counter mode offset in the srtp_protect() and srtp_unprotect()
-   functions, if that cipher is used.  This change cascaed through
-   many other functions, including srtp_init_from_hex(),
-   srtp_sender_init() and srtp_receiver_init() in rtp.c, and also
-   changing the CLI to test/rtpw.  In the future, another function
-   call will be added to the library that enables multiple ssrc/key
-   pairs to be installed into the same srtp session, so that libsrtp
-   works with multiple srtp senders.  For now, this functionality is
-   lacking.
-
-   Removed the GDOI interface to the rtpw demo program.  This will be
-   added again at a later date, after the SRTP and GDOI distributions
-   stabilize.  For now, I've left in the GDOI #defines and autoconf
-   definitions so that they'll be in place when needed.
-
-   Updated tmmhv2_compute() so that it didn't assume any particular
-   alginment of the output tag.
-
-   Changed bit field variables in srtp.h to unsigned char from
-   unsigned int in order to avoid a potential endianness issue.
-
-   Fixed rdbx_estimate_index() to handle all input cases.  This solves
-   the now notorious "abaft" bug in the rtpw demo app on linux/intel,
-   in which spurious replay protection failures happen after that word
-   is received.
-
-   Added ntohs(hdr->seq) to srtp_protect and srtp_unprotect, removed
-   from rijndael_icm_set_segment().
-
-   Added error checking and handling to srtp_sender_init() and
-   srtp_receiver_init().
-
-   Changed srtp_alloc() so that it does what you'd expect: allocate an
-   srtp_ctx_t structure.  This hides the library internals.
-
-
-1.0.1   Many minor fixes
-
-   Added cipher_driver_buffer_test(...) to test/cipher-driver.c.  This
-   function checks that the byte-buffering functions used by a cipher
-   are correct.
-
-   Fixed SunOS/Solaris build problems: added HAVE_SYS_INT_TYPES_H and
-   changed index_t to xtd_seq_num_t (see include/rdbx.h).
-
-   Fixed SEAL3.0 output byte buffering, added byte-buffering test to
-   cipher/cipher-driver.c.
-
-   Fixed roc-driver so that the non-sequential insertion test
-   automatically recovers from bad estimates.  This was required to
-   prevent spurious failures.
-
-   Made rdbx_estimate_index(...) function smarter, so that initial RTP
-   sequence numbers greater than 32,768 don't cause it to estimate the
-   rollover counter of 0xffffffff.
-
-
-1.0.0   Initial release
+  PR 263 - Addition of CUTest unit testing framework
+  PR 258 - Add AES-GCM to DTLS-SRTP Protection Profiles
+  PR 248 - Updated README now used to generate documentation
+  PR 239 - octet_string_set_to_zero() delegates to OPENSSL_cleanse() if
+           available, if not it will use srtp_cleanse() to zero memory
 
+  Lots of minor changes. See the pull requests for details.
```

Comment:
Cool! Thanks. I will go through them and do a short tagline per PR (so basically same as Other changes, which is a fine name for the list I'd say.).